### DISCUSSION TITLE
ImportError: DLL load failed while importing _greenlet: Le module spécifié est introuvable.

### DISCUSSION THREAD

**Discussion by badouzeum** [brentvollebregt/auto-py-to-exe#417] [Q&A] [answered]
I want to convert a python .py file into an .exe executable with the python module auto py to exe but when I try to open the "auto py to exe" library I get the following message:

File "C:\Users\PRECISION\AppData\Roaming\Python\Python311\site-packages\greenlet\__init__.py", line 29, in <module>
    from ._greenlet import _C_API # pylint:disable=no-name-in-module
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ImportError: DLL load failed while importing _greenlet: Le module spécifié est introuvable.

**Comment by brentvollebregt** [comment, marked as answer]
Copy of #415 (which is a duplicate of #325) - take a look at [this solution](https://github.com/brentvollebregt/auto-py-to-exe/issues/325#issuecomment-1313939303).
